13/umbraco-deploy/release-notes.md

```diff
@@ -20,33 +20,71 @@ Under each major version, you can find details about minor and patch releases fo
 
 <summary>Version 13</summary>
 
-[**13.0.0-rc3**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.0)
-(November 28th 2023)
+[**13.0.1**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.1) **(December 21th 2023)**
```

Comment:
all the release notes below have been added in PR https://github.com/umbraco/UmbracoDocs/pull/5750